src/styles/star.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  text-decoration: none;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+
+@for $starsStatus from 1 through 5 {
+  &--active-#{$starsStatus} .stars__star:nth-child(-n + #{$starsStatus}) {
+    background-image: url('../images/star-active.svg');
+  }
+}
```

```suggestion
  @for $starsStatus from 1 through 5 {
    &--active-#{$starsStatus} .stars__star:nth-child(-n + #{$starsStatus}) {
      background-image: url('../images/star-active.svg');
    }
  }
```
Remember about correct indentation

---

i`ll remember